setup.py

```diff
@@ -69,5 +69,6 @@ def finalize_options(self):
     cffi_modules=[
         'libqtile/pango_ffi_build.py:pango_ffi',
         'libqtile/core/xcursors_ffi_build.py:xcursors_ffi',
+        'libqtile/widget/pulseaudio_ffi.py:pulseaudio_ffi',
```

Comment:
I worry here about adding a pulse dependency which is only used for this widget. Is there any way to make this optional?

Reply:
I am actually not sure whether there is such a way except maybe compile it during regular qtile run. at least travis needs it to be included so that mypy wont complain. 
and then for a regular user that would install qtile - as far as I am aware qtile only ships libqtile to destination meaning there is no script that could be run in order to compile.

Reply:
What about just appending it if pkgconfig says it's present?